ui/app/css/itcss/components/new-account.scss

```diff
@@ -4,6 +4,12 @@
   box-shadow: 0 0 7px 0 rgba(0,0,0,0.08);
   z-index: 25;
   height: unset;
+  overflow-y: scroll;
```

Comment:
looking into see if there is a way to get the right behavior without doing this as we should just use the browser's default scrolling mechanism.

Reply:
Could we experiment with removing the hardcoded height from new-external-account-form? 

i get the same basic behavior you're going for if I remove these changes and do this:

![image](https://user-images.githubusercontent.com/4448075/83168247-cfebbd00-a0d6-11ea-938b-551aaabbfb7f.png)

Reply:
We had intentionally hidden the scrollbar on Chrome. I think @rekmarks was suggesting we do the same for Firefox to make it consistent.

I too would prefer that we use the default browser scrollbar where possible. Though the case could be made to consider the popup UI as a special case. The scrollbar looks pretty bad in the popup UI on some platforms, and consumes the very limited space we have to work with.

Reply:
Ah, that's a good point though - this does seem like the wrong place for this rule 🤔

Reply:
I thought we hid the scrollbar in the popup as a matter of policy.

Reply:
you can apply scrollbar-width: none to popup, just don't mess with overflow rules because you end up with 22 places you have to restore the default behavior to reduce jank.

![image](https://user-images.githubusercontent.com/4448075/83172237-e8f76c80-a0dc-11ea-8d24-bf4dad87c426.png)

My main argument is that by adding overflow-y scroll to a div you're making that div scroll independently of the browser and this can result in poor performance.